eventuate-local-java-micronaut-jdbc/src/main/java/io/eventuate/local/java/micronaut/jdbc/jdbckafkastore/EventuateLocalFactory.java

```diff
@@ -52,8 +53,11 @@ public AggregateCrud asyncAggregateCrud(io.eventuate.javaclient.commonimpl.sync.
 
   @Singleton
   public EventuateKafkaAggregateSubscriptions aggregateEvents(EventuateKafkaConfigurationProperties eventuateLocalAggregateStoreConfiguration,
-                                                              EventuateKafkaConsumerConfigurationProperties eventuateKafkaConsumerConfigurationProperties) {
-    return new EventuateKafkaAggregateSubscriptions(eventuateLocalAggregateStoreConfiguration, eventuateKafkaConsumerConfigurationProperties);
+                                                              EventuateKafkaConsumerConfigurationProperties eventuateKafkaConsumerConfigurationProperties,
+                                                              KafkaConsumerFactory kafkaConsumerFactory) {
+    return new EventuateKafkaAggregateSubscriptions(eventuateLocalAggregateStoreConfiguration,
+            eventuateKafkaConsumerConfigurationProperties,
+            kafkaConsumerFactory);
```

Comment:
What's the purpose of this change? Seems unrelated to classloader issue.

Reply:
Chris, now kafka messaging project contains this change: https://github.com/eventuate-foundation/eventuate-messaging-kafka/pull/35/files
It makes necessary to have KafkaConsumerFactory to created consumers in eventuate local and tram.
And when I tested changes I also updated eventuate-messaging-kafka project with snapshot of eventuate-common
to avoid any conflicts between versions of eventuate-common in eventuate local, eventuate tram, eventuate tram sagas.
Anyway we need to update all libraries step by step and this change is required.
Please let me remind you how dependencies look like:
eventuate-common->eventuate-common-messaging->eventuate-messaging-kafka

eventuate-common->eventuate-local
eventuate-messaging-kafka->eventuate-local

eventuate-common->eventuate-tram-core
eventuate-messaging-kafka->eventuate-tram-core

eventuate-common->eventuate-tram-sagas
eventuate-messaging-kafka->eventuate-tram-sagas

Reply:
Ok.  But the PR description just says that it's about class loading. Please update the description.